CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 ## Next release
 
+## 0.42.0 / 2020-09-09
+
+Most importantly the Prometheus Operator now lives in its own indepent GitHub organization.
+
+* [CHANGE] Introduce initial governance (#3398)
+* [FEATURE] Move API types out into their own module (#3395)
+* [FEATURE] Create a monitoring mixin for prometheus-operator (#3333)
+* [ENHANCEMENT] Instrument client-go requests (#3465)
+* [ENHANCEMENT] pkg/prometheus: skip invalid service monitors (#3445)
+* [ENHANCEMENT] pkg/alertmanager: Use lower value for --cluster.reconnect-timeout (#3436)
+* [ENHANCEMENT] pkg/alertmanager: cleanup resources via OwnerReferences (#3423)
+* [ENHANCEMENT] Add prometheus_operator_reconcile_operations_total metric (#3415)
+* [ENHANCEMENT] pkg/operator/image.go: Adjust image path building (#3392)
+* [ENHANCEMENT] Specify timeouts per Alertmanager target when sending alerts. (#3385)
+* [ENHANCEMENT] Push container images to Docker Hub and Quay (#3390)
```

Comment:
Hmm I don't think the image is on DockerHub.

Reply:
Right. I didn't get that it seems. Here's what I rephrased to:
> Push container images to Quay into coreos and prometheus-operator orgs